### DISCUSSION TITLE
GTKWave integration

### DISCUSSION THREAD

**Discussion by hasan12977** [FPGAwars/icestudio#810] [Ideas]
hello. how hard would it be to integrate GTKWave with the app so it can be used as a one standalone app? I really like this app and I want to see more features.

**Comment by cavearr** [comment]
Hi! we are working in some integrations, soon we announce as wip versoin the new featuers.

Thanks for using iceStudio and give us your feedback.

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
